docs/source/tutorials/qubo_problem.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>typo: 'is generally considered NP hard' -&gt; 'is NP hard'</p><p>typo: 'we wish to minimise the cost function int he form of' -&gt; 'we wish to minimise a cost function of the form'</p><p> </p><p>The term 'ansatz' is mentioned a few times but isn't really defined </p><p> </p><p>typo: 'this cost fucntion' -&gt; 'this cost function'</p><p>typo: 'Below, the code of solving QUBO problem is shown' -&gt; 'Below, the code used to solve this QUBO problem is shown.'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>